### PR TITLE
Move security review to the top of the list.

### PR DESCRIPTION
**Pull request by fmarier**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -3,13 +3,13 @@ Resolves
 
 ## Submitter Checklist:
 
+- [ ] Requested a security/privacy review as needed
```
Comment (diracdeltas) [inline .github/PULL_REQUEST_TEMPLATE.md:6]: minor suggestion: `as needed` should link to https://github.com/brave/handbook/blob/master/development/security.md#when-is-a-security-review-needed

Comment (fmarier) [reply .github/PULL_REQUEST_TEMPLATE.md:6]: @diracdeltas `brave/handbook` is not public, do we want to link to that in the PR template?

Comment (fmarier) [reply .github/PULL_REQUEST_TEMPLATE.md:6]: We could probably move/copy the first part of that section to the public `brave/brave-browser` wiki.

Comment (iefremov) [inline .github/PULL_REQUEST_TEMPLATE.md:6]: also https://github.com/brave/brave-browser/blob/master/.github/PULL_REQUEST_TEMPLATE.md

**Comment by fmarier** [comment]
I added links to the relevant places and rephrased this line following @bbondy 's suggestion.

**Review by diracdeltas — APPROVED** [review]

**Pull request merged**